### DISCUSSION TITLE
frequend fetch speed to localhost slow?

### DISCUSSION THREAD

**Discussion by slauko** [r0x0r/pywebview#1203] [General]
hey👋,
just wanted to ask if it is normal, that when lets say i have a react frontend with pywebview running and fetching to my backend over the js_api it takes about 100ms for the result to appear?

even when i fetch to localhost,  
im also instantly return from my api function without any calculations  :/

**Comment by r0x0r** [comment]
What pywebview version and renderer does this occur on?

**Reply by slauko** [reply]
pywebview 4.2.2
[pywebview] Using WinForms / Chromium
^ if u mean that for renderer

**Reply by r0x0r** [reply]
There was a default delay of 100ms for check return value of JS API. It is now changed it to 1ms, so in my tests JS API calls take now several milliseconds (with no heavy calculations). The fix is in master
Thanks for pointing this issue out.
